Header header
float64 pivot_error